Book/php7/internal_types/strings/smart_str.rst

```diff
@@ -70,6 +70,21 @@ parameter you'll use::
     /* Don't forget to release/free it */
     smart_str_free(&my_str);
 
+We can also use the embedded ``zend_string`` independently of the ``smart_str``::
```

Comment:
```suggestion
It is also possible to embedded the ``zend_string`` independently of the ``smart_str``::
```
Although I don't know if there is a specific style guide about not using "we"

Reply:
There are already 4 occurrences of "we" on that page, so I guess it is good as is.